src/test/java/seedu/realodex/logic/commands/CommandTestUtil.java

```diff
@@ -28,8 +28,11 @@
  */
 public class CommandTestUtil {
 
-    public static final String VALID_NAME_AMY = "Amy Bee";
-    public static final String VALID_NAME_BOB = "Bob Choo";
+    public static final String VALID_NAME_AMY_CAPS = "Amy Bee";
```

Comment:
"CAPS" seems so suggest that the name should be in caps but its not? maybe replace with "AMY BEE"
```suggestion
    public static final String VALID_NAME_AMY_CAPS = "AMY BEE";
```

Reply:
I will change to `VALID_NAME_AMY_FIRST_LETTER_CAPS`